how_to_build.md

```diff
@@ -11,41 +11,38 @@ At the time of writing the full build (including all target frameworks) can only
+The build requires that a few pieces of software be installed on the host computer. We're somewhat aggressive about adopting new language features and the like, so rather than specifying exactly which versions are required, we'll tend toward "latest" or "at least" forms of guidance. If it seems you have an incompatible version of the software, prefer to upgrade rather than downgrade.
 
-### On Windows
+FakeItEasy can be built and tested on Windows and Linux operating systems, with few differences between the two.
 
-The default [build profile](#building-only-a-subset-of-the-supported-target-frameworks) on Windows builds FakeItEasy for .NET Framework 4.6.2, .NET Standard 2.0, .NET Standard 2.1, and .NET 6.0, and runs the tests on .NET Framework 4.6.2, .NET Core 2.1, .NET Core 3.1, and .NET 6.0.
+The default [build profile](#building-only-a-subset-of-the-supported-target-frameworks) builds FakeItEasy for
+.NET 8.0,
+.NET 6.0,
+.NET Standard 2.1,
+.NET Standard 2.0,
+and .NET Framework 4.6.2 (only on Windows)
+and runs the tests on
+.NET 8.0,
+.NET 6.0,
+.NET Core 3.1,
+.NET Core 2.1,
+and .NET Framework 4.6.2 (only on Windows).
 
-Ensure that the following are installed:
 
-1. The .NET Core 2.1 and 3.1 runtimes
+Ensure that the following are installed:
 
+1. An up-to-date version of the .NET 8.0 SDK (currently this means 8.0.204 or later)
 2. The .NET 6.0 runtime
-
-3. The .NET Framework 4.6.2 or higher
-
-4. An up-to-date version of the .NET 7.0 SDK (currently this means 7.0.203 or later)
+3. The .NET Core 2.1 and 3.1 runtimes
+4. The .NET Framework 4.6.2 or higher (only on Windows)
 
 You might not need everything to run a [partial build](#building-only-a-subset-of-the-supported-target-frameworks).
 
-In order to build from Visual Studio, you will also need:
+We've only tested building FakeItEasy from Visual Studio on Windows, which requires:
 
 1. Visual Studio 2022 (17.0 or later)
 2. Install the ".NET Framework 4.6.2 targeting pack" individual component (via the Visual Studio installer)
 
-### On Linux
-
-The default [build profile](#building-only-a-subset-of-the-supported-target-frameworks) on Linux builds FakeItEasy for .NET Standard 2.0, .NET Standard 2.1, and .NET 6.0, and runs the tests on .NET Core 2.1, .NET Core 3.1, and .NET 6.0 (the .NET Framework isn't supported on Linux).
-
-Ensure the following are installed:
-
-1. The .NET Core 2.1 and 3.1 runtimes
-
-2. The .NET 6.0 runtime
-
-3. An up-to-date version of the .NET 7.0 SDK (currently this means 7.0.203 or later)
-
 ## Building
 
 Using a command prompt, navigate to your clone root folder and execute:

```

Comment:
```
PS /mnt/c/Dev/FakeItEasy> ./build.ps1

/usr/share/dotnet/sdk/7.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(144,5): error NETSDK1045: The current .NET SDK does not support targeting .NET 8.0.  Either target .NET 7.0 or lower, or use a version of the .NET SDK that supports .NET 8.0. [/mnt/c/Dev/FakeItEasy/tools/FakeItEasy.Build/FakeItEasy.Build.csproj]
```

But this is on WSL. It may be more lenient!

Reply:
Also, you're already in a PowerShell prompt 😉

Reply:
Actually, it also "works" from bash in WSL (it invokes the script, but the script fails miserably)

![image](https://github.com/FakeItEasy/FakeItEasy/assets/1864281/a8f2fa6f-fe0f-4ddd-b002-cc626a9349c8)

Reply:
Sent a PR (#2017) to fix the build.ps1 issue, so you can ignore this comment.

Reply:
Amazing. Thank you.